tools/integration_tests/run_tests_mounted_directory.sh

```diff
@@ -272,3 +272,11 @@ sudo umount $MOUNT_DIR
 gcsfuse --implicit-dirs --enable-storage-client-library=false $TEST_BUCKET_NAME $MOUNT_DIR
 GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/write_large_files/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR
 sudo umount $MOUNT_DIR
+
+gcsfuse --implicit-dirs $TEST_BUCKET_NAME $MOUNT_DIR
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/gzip/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR --testbucket=$TEST_BUCKET_NAME
+sudo umount $MOUNT_DIR
+
+gcsfuse --implicit-dirs --enable-storage-client-library=false $TEST_BUCKET_NAME $MOUNT_DIR
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/gzip/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR --testbucket=$TEST_BUCKET_NAME
+sudo umount $MOUNT_DIR
```

Comment:
Left one line at the end.

Reply:
I don't get this comment. There is no line at the end.

Reply:
I mean cursor should be in new line in the file.

Reply:
Done.